package.json

```diff
@@ -1842,8 +1842,8 @@
                 {
                     "type": "webview",
                     "id": "jupyterViewVariables",
-                    "name": "Variables",
-                    "when": "jupyter.isvscodenotebookactive"
+                    "name": "Jupyter: Variables",
```

Comment:
Per comment above if we go with this we can just change to "Jupyter Variables" ":" was added by the view container / view separation.